src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
```

Comment:
The function should handle cases where `sourceString` might be null or undefined. Consider adding a check at the beginning of the function to return an empty object if `sourceString` is falsy.